armsrc/Makefile

```diff
@@ -106,12 +106,9 @@ ARMSRC = fpgaloader.c \
 	parity.c \
 	usb_cdc.c \
 	cmd.c \
-	lf_samyrun.c \
-    vtsend.c 
-	#	lf_samyrun.c \
-	#	lf_hidbrute.c \
-	#	lf_proxbrute.c \
-	#	hf_mattyrun.c \
```

Comment:
Please don't  remove these. They are there in order to make it easy for ppl to compile different standalone modes.

Reply:
sorry. I made a PR which cleans up the makefile when it comes to standalone modes.  You will find it easier.